app/import_functions_test.go

```diff
@@ -2408,6 +2409,39 @@ func TestImportimportMultiplePostLines(t *testing.T) {
 
 	AssertAllPostsCount(t, th.App, initialPostCount, 11, team.Id)
 
+	// Create new reply with type and edit_at for existing post with replies.
+	data = LineImportWorkerData{
+		LineImportData{
+			Post: &PostImportData{
+				Team:     &teamName,
+				Channel:  &channelName,
+				User:     &user2.Username,
+				Message:  ptrStr("Message with reply"),
+				CreateAt: &replyPostTime,
+				Replies: &[]ReplyImportData{{
+					User:     &username,
+					Type:     ptrStr(model.PostTypeSystemGeneric),
+					Message:  ptrStr("Message reply 3"),
+					CreateAt: &replyTime,
+					EditAt:   &replyEditTime,
+				}},
+			},
+		},
+		1,
+	}
+	errLine, err = th.App.importMultiplePostLines(th.Context, []LineImportWorkerData{data}, false)
+	assert.Nil(t, err, "Expected success.")
+	assert.Equal(t, 0, errLine)
+
+	AssertAllPostsCount(t, th.App, initialPostCount, 12, team.Id)
+
+	// Check the reply values.
+	replies, nErr = th.App.Srv().Store.Post().GetPostsCreatedAt(channel.Id, replyTime)
+	assert.NoError(t, nErr, "Expected success.")
+	reply = replies[0]
+	replyBool = reply.Type != *(*data.Post.Replies)[0].Type || reply.Message != *(*data.Post.Replies)[0].Message || reply.CreateAt != *(*data.Post.Replies)[0].CreateAt || reply.EditAt != *(*data.Post.Replies)[0].EditAt || reply.UserId != user.Id
```

Comment:
I'd suggest to extract `data.Post.Replies[0]` in a separate variable to make it look a bit cleaner.